openapi/openapi.yaml

```diff
@@ -913,7 +913,10 @@ components:
                   type: number
             creator:
               type: string
-              description: user email
+              description: identical to the sub field in Cognito
+            preferred_username:
+              type: string
+              description: identical to the preferred_username field in Cognito
```

Comment:
Seems like too much detail for the API doc. We might not be using Cognito forever, so I think we can say something more generic.